website/pages/en/network/indexing.mdx

```diff
@@ -182,7 +182,7 @@ Note: To support agile scaling, it is recommended that query and indexing concer
 
 #### Create a Google Cloud Project
 
-- Clone or navigate to the Indexer repository.
+- Clone or navigate to the [Indexer repository](https://github.com/graphprotocol/indexer).
 
 - Navigate to the ./terraform directory, this is where all commands should be executed.
```

Comment:
```suggestion
- Navigate to the `./terraform` directory, this is where all commands should be executed.
```

I know you didn't change this, but I saw it right below a line you changed so why not.